moped-database/migrations/1654055457679_update_project_list_view_fund_notes_is_del/up.sql

```diff
@@ -0,0 +1,97 @@
+CREATE OR REPLACE VIEW "public"."project_list_view" AS 
```

Comment:
[Similar to the PR #656](https://github.com/cityofaustin/atd-moped/pull/656#discussion_r889060941), I would suggest a `drop view` statement before the `create`.

Reply:
This is now covered in the latest migration pushed here. 🙏